### ISSUE TITLE
transcription content should have a lang attribute in html

### ISSUE THREAD

**Issue by rlskoeser** [Princeton-CDH/geniza#1186]
## dev notes

We have this in the search results but it seems to have gotten lost somewhere along the way when we switched to the new image+transcription panel templates.

We should set `lang=` based on primary language of the document (if there is one and we have a code), or empty lang if we can't determine. This is an accessibility issue, since screen readers will attempt to read content based on the current language.

We may need a fallback attribute to set the script when known even if we don't have a language code so that we can style Hebrew and Arabic fonts properly.

**Comment by rlskoeser** [comment]
Some quick checking on languages with my local copy of production data from yesterday.

Of 32,234 total documents (31,742 public/non-suppressed) we have:
- 9,161 with no primary language
- 18,684 with one primary language
- 3,726 with two primary languages
- 636 with 3 primary languages
- 26 with 4 primary languages
- 1 with 5 primary languages

Looking at a sample of records with multiple languages, I see a number of cases where the script is common even when the language differs (e.g. Hebrew & Judaeo-Arabic or Hebrew, Arabic, & Judaeo-Arabic), so I think using the primary script for transcription font styling will be useful.

**Comment by rlskoeser** [comment]
Confirmed language codes and script attribute present in qa in search results and document details page
